src/services/iframe.js

```diff
@@ -274,7 +280,7 @@ class BaseIframeService extends G3WObject {
       PluginsRegistry
         .onafter('registerPlugin', async plugin => {
           await plugin.isReady();
-          if (plugin.getName() === this.pluginName) {
```

Comment:
@volterra79

Yoda conditions should be always written in order to avoid variable assignments. That means, if somewhere you accidentally wrote a single `=` (instead of `===`), your code should crash:

```js
plugin.getName() = this.pluginName // OK → cannot assign to function call
```

```js
this.pluginName = plugin.getName() // KO → valid variable assignment
```

Reply:
@Raruto ok